Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && \
+    apt-get install -y \
+      python \
+      python-pip \
+      libcapstone-dev \
+    && \
+    rm -fr /var/lib/apt/lists/*
+
+RUN pip install 'capstone>=3.0,<4.0'
+
+COPY . /src/github.com/xoreaxeaxeax/sandsifter
+WORKDIR /src/github.com/xoreaxeaxeax/sandsifter
```

Comment:
There's not really a need to namespace the code so thoroughly, as installing a dozen other applications or different versions would be an anti-pattern for running containers. `/sandsifter` or `/app` would suffice.